cli/template/page-studs/_app/with-trpc.tsx

```diff
@@ -40,6 +40,19 @@ export default withTRPC<AppRouter>({
        * @link https://react-query.tanstack.com/reference/QueryClient
        */
       // queryClientConfig: { defaultOptions: { queries: { staleTime: 60 } } },
+
+      // To use SSR properly you need to forward the client's headers to the server
+      /*headers: () => {
```

Comment:
This style of comment is not ECMAscript standard compliant. Can you comment the code back in and use VS Code to "comment it out"? Should result in // in front of each line

Reply:
Sure thing. On it.